src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-visibilityFilter.html

```diff
@@ -1,7 +1,15 @@
 <div>
-    The visibility filter is used to filter which warnings that are used for the conditions.
-    It matches the items under "Visible Warnings" under an analysis in CodeSonar.
-
-    Please use the ID number of the filter you want to use. You can find the ID numbers under
-    this url <b>hub_address</b>/savedsearches.html?filter=3&ssdomain=0&ssearchsgrd=Ay_0D_BTl3BTl3AgBg
+    <p dir="auto">A CodeSonar warning visibility filter that will be used by the CodeSonar plugin to determine the total list of warnings when evaluating analysis condition options.</p>
+    <ul dir="auto">
+        <li>
+        	<p dir="auto">You can specify a warning visibility filter by <em>name</em> or by numeric <em>ID</em>.</p>
+        </li>
+        <li>
+        	<p dir="auto">When left empty, it defaults to CodeSonar predefined filter named <em>active</em>.</p>
```

Comment:
By default, the <em>active</em> filter is used.

Reply:
The plugin itself assigns a default filter, right now it uses whatever filter (saved search) has the name "active".  I suppose we could change the code so that it does not specify a filter at all if this field is empty.  We chose not to do that in this release since the previous behavior was to use the filter with numeric ID 2.  Numeric IDs are not standard across hubs, so we changed the code to use the filter name instead.  We didn't want to change existing behavior in this release (other than logging).

Do you think we should change this help description further?  Should we change the behavior?

Reply:
It is possible that I misunderstood your comment.  Perhaps you just meant we should rephrase the text to say:

> By default, the _active_ filter is used.

We tried to be specific about how the plugin will filter results (i.e. based on filter name) so that users would not be surprised.

I apologize if I misunderstood!

Reply:
yes, "By default, the active filter is used." is my proposed replacement for this bullet

It would be misleading to talk about "CodeSonar predefined filter" here because the filter applied will correspond to 
 whatever saved search is currently named "active" - this will not necessarily have the same conditions that were in the factory settings.